x-pack/legacy/plugins/apm/cypress/integration/apm.spec.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+// eslint-disable-next-line spaced-comment
+/// <reference types="cypress" />
+
+import { loginAndWaitForPage } from './helpers';
+
+const rangeFrom = '2019-09-04T18:00:00.000Z';
+const rangeTo = '2019-09-05T06:00:00.000Z';
+
+describe('When clicking opbeans-go service', () => {
+  before(() => {
+    // open service overview page
+    loginAndWaitForPage(
+      `/app/apm#/services?rangeFrom=${rangeFrom}&rangeTo=${rangeTo}`
+    );
+
+    // show loading text for services
+    cy.contains('Loading...');
+
+    // click the first service in the list
+    cy.get('[data-cy=link-to-service-opbeans-go]').click({ force: true });
```

Comment:
we can also use:
```
cy.get(':contains(opbeans-go)').last()
```

Possibly that would remove the need to add `data-cy` attributes. I would like to avoid that, because I think it's going to be annoying to update a bunch of our components to support a data-cy attribute.

Reply:
We could also use the `className` property which is supported already by many of our components.